client/my-sites/customer-home/locations/primary/index.jsx

```diff
@@ -11,19 +12,28 @@ import Webinars from 'my-sites/customer-home/cards/tasks/webinars';
 import FindDomain from 'my-sites/customer-home/cards/tasks/find-domain';
 import SiteSetupList from 'my-sites/customer-home/cards/tasks/site-setup-list';
 import { PerformanceTrackerStop } from 'lib/performance-tracking';
+import GoMobile from 'my-sites/customer-home/cards/features/go-mobile';
 
 const cardComponents = {
 	'home-task-site-setup-checklist': SiteSetupList,
 	'home-task-connect-accounts': ConnectAccounts,
 	'home-task-find-domain': FindDomain,
 	'home-task-webinars': Webinars,
+	'home-feature-go-mobile-on-mobile': GoMobile,
```

Comment:
D43129-code updates the API to return a `home-task-go-mobile` card identifier.
```suggestion
	'home-task-go-mobile': GoMobile,
```